src/Routes.jsx

```diff
@@ -22,11 +26,26 @@ export default function RoutesComponent() {
             path={path.productDetail}
             element={<DetailProduct />}
           />
+          <Route path={path.notFound} element={<NotFound />} />
           <Route element={<AuthenticatedGuard />}>
             <Route path={path.cart} element={<Cart />} />
             <Route path={path.payment} element={<Payment />} />
+            <Route element={<User />}>
```

Comment:
```suggestion
            <Route path={path.user} element={<User />}>
```
Như này thì mấy cái ở dưới em k cần phải `path.user +` nữa

Reply:
em đã fix